File: Bean.API/Bean.POCO/Family.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Runtime.Serialization;

namespace Bean.POCO
{
    [DataContract(IsReference = true)]
    public class Family : BasePOCO
    {
        [DataMember]
        [Key]
        public int Id { get; set; }

        [DataMember]
        public int? BinderId { get; set; }

        [DataMember]
        [ForeignKey("BinderId")]
        public Binder Binder { get; set; }

        [DataMember]
        [Required]
        [MaxLength(50)]
        public string Name { get; set; }

        [DataMember]
        public List<Plant> Plants { get; set; }

        [DataMember]
        [Required]
        public Status Status { get; set; }

    }
}

File: Bean.Client/app/families/family.ts
import { IBaseObject } from '../shared/base';
import { Status } from '../shared/enums';
import { Binder } from '../binders/binder';
import { Plant } from '../plants/plant';

interface IFamily {
    id: number;
    binderId?: number;
    binder: Binder;
    name: string;
    plants: Plant[];
    status: Status;
}

export class Family implements IFamily, IBaseObject {

    isNew = true;

    id = 0;
    binderId = 0;
    binder = new Binder();
    name = "";
    plants = Plant[0];
    status = Status.Enabled;
}